app/main.py

```diff
@@ -1,3 +1,19 @@
-def cache(func):
-    # Write your code here
-    pass
+from typing import Callable, Any
+
+
+def cache(func: Callable) -> Callable:
+    cache_dict = {}
+
+    def inner(*args) -> Any:
```

Comment:
Your function returns not any, but Callable

Reply:
but wht not `Any ` if  `return` cache_dict[args]` from inner can be int, list, str, etc. (depends on the type of *args)